cli/args/flags.rs

```diff
@@ -1900,6 +1906,31 @@ fn clean_subcommand() -> Command {
     cstr!("Remove the cache directory (<c>$DENO_DIR</>)"),
     UnstableArgsConfig::None,
   )
+  .defer(|cmd| {
+    cmd
+      .arg(
+        Arg::new("except_paths")
```

Comment:
This is a nitpick, but we generally use dashes in flag names too. Not a big deal though, it's not enforced in any way